cmd/clef/tutorial.md

```diff
@@ -134,13 +142,15 @@ function ApproveSignData(r){
 }
 
 ```
-In this example,
-* any requests to sign data with the account `0x694...` will be
-    * auto-approved if the message contains with `bazonk`,
-    * and auto-rejected if it does not.
-    * Any other signing-requests will be passed along for manual approve/reject.
+In this example:
+* Any requests to sign data with the account `0x694...` will be
+    * auto-approved if the message contains with `bazonk`
+    * auto-rejected if it does not.
+* Any other signing-requests will be passed along for manual approve/reject.
 
-..attest the new file
+_Note: make sure that `0x694...` is an account you have access to. You can create it either via the clef or the traditional account cli tool. If the latter was chosen, make sure both clef and geth use the same data directory by specifing `--networkid your_network_id` and `--keystore path/to/your/keystore` when running clef._
```

Comment:
`--networkid` is normally not needed, since  `--keystore` can be used to point directly to the keystore. If a separate testnet-based network is used, then I guess the user can use `--networkid` to change what the default location is where clef searches for the keystore. 
However, `--networkid` will also change the transaction semantics, to add replay protection. So it's useful, but has nothing really to do with finding the right data directory, other than implicitly so.